src/store/simpleCartReducer.js

```diff
@@ -0,0 +1,44 @@
+//categories reducer
+
+let initialState = {
+  simpleCart:[]
+};
+
+//actions
+export const addToSimpleCart = (product)=>{
+  // console.log('product in simplecart: ', product)
+  return{
+    type: 'ADD',
+    payload: product,
+  }
+}
+
+export const removeFromSimpleCart = (product)=>{
+  // console.log('product in simplecart: ', product)
+  return{
+    type: 'REMOVE',
+    payload: product,
+  }
+}
+
+const simpleCartReducer = (state=initialState, action)=>{
+  let {type, payload} = action;
+  switch(type){
+    case 'ADD':
+      // do case things
+      // state.simpleCart.name !== payload ? state.simpleCart.push(payload) : state.simpleCart.inventoryCount +1;
+      state.simpleCart.push(payload)
+      return{...state}
+
+    case 'REMOVE':
+      // do case things
+      // state.simpleCart.push(payload);
+      return{...state}
+    
+
+    default:
+      return state;
+  }
+}
+
+export default simpleCartReducer;
```

Comment:
-1 point for lack of styling
-.5 point because your product should have a reducer on stock # when it's added to the cart
-.5 point for not having a cart on the top of the page that displays number of items in it
-1 point for a lack of README
-1 point for lack of header and footer